cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -117,6 +117,14 @@
       <artifactId>twitter4j-stream</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-mapreduce-client-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
```

Comment:
yeah, so without this, I had no way of testing with mysql. We need to package the jar with the ETL adapter somehow.